### ISSUE TITLE
Recurrent coughing spasms

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6482]
**Preferred term label:**
Recurrent coughing spasms

**Synonyms**

**Definition (free text, please give PubMed ID)**
Repeated occurrence of episodes of coughing, with each episode consisting of at least five minutes of continuous coughing.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012735

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Charcot-Marie-Tooth disease, type 2J
https://omim.org/entry/607736
https://pubmed.ncbi.nlm.nih.gov/15159512/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
